rails/locale/pt.yml

```diff
@@ -63,41 +63,41 @@ pt:
   datetime:
     distance_in_words:
       about_x_hours:
-        one: aproximadamente 1 hora
+        one: aproximadamente %{count} hora
         other: aproximadamente %{count} horas
       about_x_months:
-        one: aproximadamente 1 mês
+        one: aproximadamente %{count} mês
         other: aproximadamente %{count} meses
       about_x_years:
-        one: aproximadamente 1 ano
+        one: aproximadamente %{count} ano
         other: aproximadamente %{count} anos
       almost_x_years:
-        one: quase 1 ano
+        one: quase %{count} ano
         other: quase %{count} anos
       half_a_minute: meio minuto
       less_than_x_seconds:
-        one: menos de 1 segundo
+        one: menos de %{count} segundo
         other: menos de %{count} segundos
       less_than_x_minutes:
         one: menos de um minuto
```

Comment:
You missed this one:

one: menos de %{count} minuto

Reply:
Good call. I was just using a very simple regex to find these.

I've now written a script to detect differing interpolations between the pluralization context subkeys. Running it, I see that there are 566 similar issues across 71 locales:

```
16 problems in `ar` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.other_than", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
11 problems in `bn` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
6 problems in `bs` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
12 problems in `cs` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
13 problems in `da` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
13 problems in `de` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
12 problems in `de-AT` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
12 problems in `de-CH` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
12 problems in `de-DE` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
15 problems in `dz` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
7 problems in `el` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
7 problems in `el-CY` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
1 problems in `en` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-AU` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-CA` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-CY` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-GB` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-IE` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-IN` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-NZ` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-TT` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-US` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `en-ZA` locale: ["datetime.distance_in_words.less_than_x_minutes"]
7 problems in `eo` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
2 problems in `et` locale: ["datetime.distance_in_words.almost_x_years", "errors.template.header"]
11 problems in `eu` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
11 problems in `fa` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months"]
13 problems in `fi` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
10 problems in `fr` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
10 problems in `fr-CA` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
10 problems in `fr-CH` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
10 problems in `fr-FR` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
16 problems in `fy` locale: ["datetime.distance_in_words.about_x_hours.one", "datetime.distance_in_words.about_x_months.one", "datetime.distance_in_words.about_x_years.one", "datetime.distance_in_words.almost_x_years.one", "datetime.distance_in_words.less_than_x_seconds.one", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years.one", "datetime.distance_in_words.x_seconds.one", "datetime.distance_in_words.x_minutes.one", "datetime.distance_in_words.x_days.one", "datetime.distance_in_words.x_months.one", "datetime.distance_in_words.x_years.one", "errors.messages.too_long.one", "errors.messages.too_short.one", "errors.messages.wrong_length.one", "errors.template.header.one"]
2 problems in `gl` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.over_x_years"]
12 problems in `he` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
10 problems in `hi` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
10 problems in `hi-IN` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
9 problems in `id` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months"]
1 problems in `is` locale: ["errors.template.header"]
7 problems in `it` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
6 problems in `it-CH` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
7 problems in `kn` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
8 problems in `lb` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.wrong_length"]
10 problems in `mg` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
4 problems in `ml` locale: ["datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.template.header"]
13 problems in `mr-IN` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
2 problems in `ms` locale: ["datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes"]
1 problems in `nb` locale: ["errors.template.header"]
7 problems in `nl` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years"]
15 problems in `oc` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
11 problems in `pl` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
1 problems in `pt` locale: ["datetime.distance_in_words.less_than_x_minutes"]
1 problems in `pt-BR` locale: ["datetime.distance_in_words.less_than_x_minutes"]
9 problems in `rm` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months"]
7 problems in `ro` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.template.header"]
16 problems in `sc` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds.one", "datetime.distance_in_words.x_minutes.one", "datetime.distance_in_words.x_days.one", "datetime.distance_in_words.x_months.one", "datetime.distance_in_words.x_years.one", "errors.messages.too_long.one", "errors.messages.too_short.one", "errors.messages.wrong_length.one", "errors.template.header.one"]
12 problems in `sk` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
2 problems in `sl` locale: ["datetime.distance_in_words.less_than_x_minutes", "errors.template.header"]
1 problems in `sq` locale: ["datetime.distance_in_words.less_than_x_minutes"]
16 problems in `st` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
13 problems in `sv` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
12 problems in `sv-FI` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
13 problems in `sv-SE` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.template.header"]
3 problems in `sw` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.almost_x_years", "errors.template.header"]
5 problems in `ta` locale: ["datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "errors.messages.too_long", "errors.messages.too_short"]
16 problems in `te` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "datetime.distance_in_words.x_years", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
12 problems in `tl` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.template.header"]
14 problems in `tt` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length"]
15 problems in `ug` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
15 problems in `ur` locale: ["datetime.distance_in_words.about_x_hours", "datetime.distance_in_words.about_x_months", "datetime.distance_in_words.about_x_years", "datetime.distance_in_words.almost_x_years", "datetime.distance_in_words.less_than_x_seconds", "datetime.distance_in_words.less_than_x_minutes", "datetime.distance_in_words.over_x_years", "datetime.distance_in_words.x_seconds", "datetime.distance_in_words.x_minutes", "datetime.distance_in_words.x_days", "datetime.distance_in_words.x_months", "errors.messages.too_long", "errors.messages.too_short", "errors.messages.wrong_length", "errors.template.header"]
1 problems in `wo` locale: ["datetime.distance_in_words.less_than_x_minutes"]
Total: 566 problems in 71 locales
```